descartes_light/CMakeLists.txt

```diff
@@ -54,19 +75,24 @@ target_include_directories(${PROJECT_NAME} SYSTEM PUBLIC
 # Declare a C++ library
 add_library(${PROJECT_NAME}_gantry SHARED src/gantry_kinematics.cpp)
 target_link_libraries(${PROJECT_NAME}_gantry PUBLIC console_bridge::console_bridge ${PROJECT_NAME}_core)
+target_cxx_version(${PROJECT_NAME}_gantry PUBLIC VERSION 11)
 descartes_target_compile_options(${PROJECT_NAME}_gantry PUBLIC)
+if(NOT MSVC)
+  target_clang_tidy(${PROJECT_NAME}_gantry ENABLE ${DESCARTES_ENABLE_TESTING} ARGUMENTS ${CLANG_ARGS})
+  target_code_coverage(${PROJECT_NAME}_gantry ALL ENABLE ${DESCARTES_ENABLE_TESTING})
+endif()
 target_include_directories(${PROJECT_NAME}_gantry PUBLIC
     "$<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}/include>"
     "$<INSTALL_INTERFACE:include>")
 target_include_directories(${PROJECT_NAME}_gantry SYSTEM PUBLIC
   ${EIGEN3_INCLUDE_DIRS})
 
-descartes_configure_package(${PROJECT_NAME}_core ${PROJECT_NAME} ${PROJECT_NAME}_gantry)
+configure_package(NAMESPACE descartes TARGETS ${PROJECT_NAME}_core ${PROJECT_NAME} ${PROJECT_NAME}_gantry)
 
 # Mark cpp header files for installation
 install(DIRECTORY include/${PROJECT_NAME}
   DESTINATION include
-  FILES_MATCHING PATTERN "*.h*"
+  FILES_MATCHING PATTERN "*.h" PATTERN "*.hpp" PATTERN "*.hxx"
   PATTERN ".svn" EXCLUDE
```

Comment:
We should just remove all these lines; I don't think there are any files in the `include` directory that we don't want to install, and adding the matching patterns will eventually result the exclusion of file whose extensions we forget to add here

Reply:
Done